pkg/controllers/management/auth/prtb_handler.go

```diff
@@ -36,6 +36,7 @@ var projectManagmentPlaneResources = []string{
 }
 var prtbClusterManagmentPlaneResources = []string{
 	"notifiers",
+	"clustercatalogs",
```

Comment:
will the user only be able to list the cluster catalog with this change? I want to make sure only list/watch is allowed similar to https://github.com/rancher/rancher/pull/17896/files#diff-80f1d22b644f4a6ca5de869a25a292f2R50

Reply:
For project-member and project-owner roles, we have the verbs set to “get”,“list”,“watch” for clustercatalogs
https://github.com/rancher/rancher/blob/master/app/role_data.go#L187
https://github.com/rancher/rancher/blob/master/app/role_data.go#L166

so the method `grantManagementProjectScopedPrivilegesInClusterNamespace` will use these roles with the exact verbs for the prtbs while create rolebindings. But I'll add another test to make sure create/update/delete of clustercatalogs isn't allowed by project roles